### DISCUSSION TITLE
@ant-design/pro-cli 和 create-umi 有什么区别

### DISCUSSION THREAD

**Discussion by chentianxin** [umijs/umi#10614] [Q&A]
### 🧐 问题描述

pro-cli 和 create-umi 都能创建 antd-pro 项目，他们有什么区别呢，我应该用哪个或者推荐用哪个呢？

### 💻 示例代码
```sh
npm i @ant-design/pro-cli -g
pro create myapp
// 或者
pnpm dlx create-umi@latest
```

**Comment by fz6m** [comment]
`@ant-design/pro-cli` 创建出来的模板项目更复杂，有登录页等等，create-umi 创建出来的更简约，你可以创建对比看看。
